docs/co-reference_resolution.md

```diff
@@ -23,6 +23,8 @@ Average of F1-scores returned by these three precision/recall metrics:
 - MUC.  
 - B-cubed.  
 - Entity-based CEAF.  
+- BLANC.
+- the link based entity aware LEA metric.
```

Comment:
One minor suggestion, could you modify line 27 into `- Link-Based Entity-Aware metric (LEA).` ? This is a more official name based on the original [paper](https://www.aclweb.org/anthology/P16-1060.pdf).

Reply:
of course:)